workflow-templates/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Mark and Close stale issues'
+
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'Marking issue as stale, since there has been no activity in 30 days. This issue will be closed unless it is updated or the "stale" label is removed.'
```

Comment:
This is ambiguous. If I'm interpreting it correctly, after 30 days of inactivity, the issue is automatically closed and labeled "stale". If anyone comments on the issue or removes the "stale" label, it is automatically re-opened. Is this the correct interpretation?

Reply:
After 30 days of inactivity the issue is just labeled stale, I disabled automatically closing. 

The thinking was we might want a chance to review stale issues before closing but if we're diligent about labeling issues with the exempt labels then we might as well enable automatic closing.

Reply:
Can we provide a 1 week review period before a stale issue is automatically closed? This way we get a chance to update or assess the issue (if there was no exempt label for example), but can also keep stale issues from piling up. Message could be: 
"Marking this issue as stale since there has been no activity in 30 days. Unless it is updated or the 'stale' label is removed, this issue will be closed within 7 days."